src/App.css

```diff
@@ -36,3 +36,8 @@
     transform: rotate(360deg);
   }
 }
+
+.error-txt {
+  color: red;
```

Comment:
If you'd like to have a consistent color palette, try checking out https://www.designspiration.com/. You can select an image or color scheme you like and it will give you a detailed color palette that matches. This can help ensure your site has consistent and cohesive colors.